com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/MaxRectsLayoutStrategy.java

```diff
@@ -70,7 +70,8 @@ public List<Layout> createLayout(List<Rect> srcRects) {
             ArrayList<Rect> rects = new ArrayList<Rect>(page.outputRects.size());
             for(RectNode node : page.outputRects) {
                 Rect finalRect = new Rect(node.rect.getId(), node.rect.getIndex(), node.rect.getPage(),
-                                           node.rect.getX(), node.rect.getY(), node.rect.getWidth() - settings.paddingX, node.rect.getHeight() - settings.paddingY,
+                                           node.rect.getX() + settings.paddingX, node.rect.getY() + settings.paddingY,
+                                           node.rect.getWidth() - settings.paddingX, node.rect.getHeight() - settings.paddingY,
```

Comment:
In line 55-56 we add `settings.paddingX()` and `settings.paddingY` to each image rect to take it into account. Here we subtract it from rect size to restore size as it was.
This operation add margin between images + on the right and bottom edge of the atlas (because it affects rect size and 0;0 is  in top left corner), but left and top edges of the atlas ignored.

To take it into account I reduce size of the atlas for margin (`settings.paddingX` and `settings.paddingY`) and here I move each rect position to add this space from left and top side.